### PR TITLE
Added a /stop command

### PR DESCRIPTION
**Pull request by ghost**
I wanted to help and add a few commands

### CODE REVIEW

File: src/com/superiornetworks/icarus/commands/Command_stop.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        ICM_Util.bcastMsg("IcarusMod is going offline.", ChatColor.YELLOW);
+
+        for (Player player : server.getOnlinePlayers())
+        {
+            player.kickPlayer("IcarusMod is going offline, come back in about 20 seconds.");
+        }
+
+        server.shutdown();
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_stop.java:21]: The variable 'server' is not defined.

**Comment by ghost** [comment]
it worked when i tested on my server

**Comment by Hockeyfan360** [comment]
@Superior-tL That's probably because a /stop command is already defined by Spigot. Besides, there's no need for this anyways.

**Comment by ghost** [comment]
Ok I was trying to help with the mod.

**Comment by mgejay** [comment]
@Superior-tL Make pull requests that solve issues, as well as actually work. You are just spamming PRs at this point

**Pull request closed without merging**